src/components/ChatLog.jsx

```diff
@@ -0,0 +1,35 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const chatEntries = props.entries.map((entry) => {
+    return (
+      <ChatEntry
+        key={entry.id}
+        id={entry.id}
+        sender={entry.sender}
+        timeStamp={entry.timeStamp}
+        body={entry.body}
+        liked={entry.liked}
+        onLike={props.onLike}
+      />
```

Comment:
Since the names of the keys on `entry` are the same as the names your are setting on `ChatEntry` attributes/you are using all of the values in `entry` you could do something something like this to save you a few keystrokes: 

```javascript
  const chatComponents = entries.map((entry) => {
    return(
      <ChatEntry
        {...entry}
        onLikeToggle={onLikeBtnToggle}
        key={entry.id}
      />
    );
  });
```

Just be mindful that this won't be as explicit as what you have, a trade off!